README.md

```diff
@@ -138,10 +138,11 @@ which should output `nvm` if the installation was successful. Please note that `
 
 If you're running a system without prepackaged binary available, which means you're going to install nodejs or io.js from its source code, you need to make sure your system has a C++ compiler. For OS X, Xcode will work, for Debian/Ubuntu based GNU/Linux, the `build-essential` and `libssl-dev` packages work.
 
-**Note:** `nvm` does not support Windows (see [#284](https://github.com/nvm-sh/nvm/issues/284)), but may work in WSL (Windows Subsystem for Linux) depending on the version of WSL. For Windows, two alternatives exist, which are neither supported nor developed by us:
+**Note:** `nvm` does not support Windows (see [#284](https://github.com/nvm-sh/nvm/issues/284)), but may work in WSL (Windows Subsystem for Linux) depending on the version of WSL. For Windows, a few alternatives exist, which are neither supported nor developed by us:
 
   - [nvm-windows](https://github.com/coreybutler/nvm-windows)
   - [nodist](https://github.com/marcelklehr/nodist)
+  - [nvs](https://github.com/pawelt/nvs)
```

Comment:
this isn't what nvs is - nvs is https://github.com/jasongin/nvs, which predates it and is much, much more used.
```suggestion
  - [nvs](https://github.com/jasongin/nvs)
```

Reply:
You are right, I have just come across https://github.com/pawelt/nvs and knew nothing about https://github.com/jasongin/nvs. I applied the suggestion 💪